napari/layers/points/_tests/test_points_utils.py

```diff
@@ -1,12 +1,29 @@
 import numpy as np
 import pandas as pd
 
-from napari.layers.points.points_utils import (
+from napari.layers import Points
+from napari.layers.points._points_utils import (
     dataframe_to_properties,
     guess_continuous,
 )
 
 
+def create_known_points_layer():
+    """Create points layer with known coordinates."""
+    data = [[1, 3], [8, 4], [10, 10], [15, 4]]
+    first_point = [1, 3]
+    known_non_point = [20, 30]
+    n_points = len(data)
+
+    layer = Points(data, size=1)
+    assert np.all(layer.data == data)
+    assert layer.ndim == 2
+    assert len(layer.data) == n_points
+    assert len(layer.selected_data) == 0
+
+    return layer, n_points, first_point, known_non_point
```

Comment:
It might be more generally useful to return `data` instead of `first_point`. I think it's fine either way though.

Reply:
i've dropped `first_point` and now use `layer.data` instead, it is more clear